app/src/main/java/com/example/harvestfresh/fragments/EventsFragment.java

```diff
@@ -67,6 +78,33 @@ public class EventsFragment extends Fragment {
         @Override
         public void onMapReady(GoogleMap googleMap) {
             mMap = googleMap;
+            mMap.setOnMarkerClickListener(new GoogleMap.OnMarkerClickListener() {
+                @Override
+                public boolean onMarkerClick(@NonNull Marker marker) {
+
+                    if (doubleClickPress) {
+                        StoreFront store = mMarkertoStore.get(marker);
+                        Intent i = new Intent(getContext(), DetailActivity.class);
+                        i.putExtra(StoreFront.class.getSimpleName(), Parcels.wrap(store));
+                        getContext().startActivity(i);
+                    } else {
+                        doubleClickPress = true;
```

Comment:
What does this do?

Reply:
Sorry for the late response, this for the doubleclick on markers